_sphinx/conf.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+#
+# rosindex documentation build configuration file, created by
+# sphinx-quickstart on Tue Oct  2 16:34:57 2018.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# sys.path.insert(0, os.path.abspath('.'))
+import sys
+import os
+import recommonmark
+from recommonmark.transform import AutoStructify
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = ['.rst', '.md']
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'rosindex'
+copyright = u'2018, Open Robotics'
+author = u'Open Robotics'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u''
+# The full version, including alpha/beta/rc tags.
+release = u''
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This patterns also effect to html_static_path and html_extra_path
+exclude_patterns = ['_build', '_scripts', '_cache', '_devel', '_plugins_data', '_ruby_libs', 'node_modules', 'js', 'Thumbs.db', '.DS_Store']
```

Comment:
@apojomovsky I don't get the rationale of this setting. Isn't sphinx running on a subdirectory of Jekyll's source dir?

Reply:
You're right, just fixed this. Thanks!